src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from django.core.exceptions import DisallowedRedirect
+from django.http import HttpResponseRedirect
+
+import requests
+from furl import furl
+from mozilla_django_oidc.views import get_next_url
+
+from digid_eherkenning_oidc_generics.mixins import (
+    SoloConfigDigiDMixin,
+    SoloConfigEHerkenningMixin,
+)
+from digid_eherkenning_oidc_generics.views import (
+    OIDCAuthenticationCallbackView as _OIDCAuthenticationCallbackView,
+    OIDCAuthenticationRequestView as _OIDCAuthenticationRequestView,
+)
+
+from ...views import BACKEND_OUTAGE_RESPONSE_PARAMETER
+from .backends import (
+    OIDCAuthenticationDigiDBackend,
+    OIDCAuthenticationEHerkenningBackend,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class OIDCAuthenticationRequestView(_OIDCAuthenticationRequestView):
+    plugin_identifier = ""
+
+    def get(self, request):
+        redirect_field_name = self.get_settings("OIDC_REDIRECT_FIELD_NAME", "next")
+        next_url = get_next_url(request, redirect_field_name)
+        if not next_url:
+            raise DisallowedRedirect
+
+        try:
+            # Verify that the identity provider endpoint can be reached
+            response = requests.head(self.OIDC_OP_AUTH_ENDPOINT)
+            if response.status_code > 400:
+                response.raise_for_status()
```

Comment:
Can't we drop the `if`? it'll raise if the status is between 400 and < 600 afaik

Reply:
I think I explicitly check for >400, because directly accessing the `OIDC_OP_AUTH_ENDPOINT` returns a 400